documentation/design-docs/dotnet-tools.md

```diff
@@ -280,6 +280,54 @@ COLLECT
 
 ```
 
+RUN (on .NET 5 or later)
+
+    Examples:
+
+    1. Launch my-server-app.exe and start monitoring it from its startup with default list of counters and interval.
+```
+    dotnet-counters run --exec C:\path\to\my-server-app.exe --mode monitor
+```
+
+    2. Launch my-server-app.exe and start collecting metrics into a CSV file from its startup with the runtime and Kestrel counters and update interval of once per minute.
+```
+    dotnet-counters run System.Runtime Microsoft-AspNetCore-Server-Kestrel --exec C:\path\to\my-server-app.exe --refresh-interval 60 --mode collect --format csv
+```
+
+    Syntax:
+
+```
+    dotnet-counters collect [-h||--help]
```

Comment:
```suggestion
    dotnet-counters run [-h||--help]
```

Reply:
I find the verb `run` to be confusing. Without reading the proposal, I'd assumed `dotnet counters run` was similar to `dotnet run` where I could point it to a directory containing a project file as opposed to needing to specify an executable